doc/releases/mimic.rst

```diff
@@ -27,6 +27,10 @@ Major Changes from Luminous
 
   * RGW can now replicate a zone (or a subset of buckets) to an
     external cloud storage service like S3.
+  * RGW now supports the S3 multi-factor authentication api on
```

Comment:
we mention this again in the major changes from luminous as well (below the doc.) though I'm ok with highlighting it here as well